source/_integrations/spokestack_wakeword.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: "Spokestack Wakeword"
+description: "Spokestack Wakeword documentation"
+ha_release: "0.38"
+ha_category: Voice
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@spokestack'
+  - '@will-rice'
+ha_domain: spokestack_wakeword
+---
+
+[Spokestack](https://www.spokestack.io/) offers open-source libraries for adding a voice interface to _anything_. `spokestack_wakeword` is an integration that enables users to add a spoken keyword
+to trigger an event.
+
+If you already have an account, [log in](https://www.spokestack.io/login), otherwise you will need to [create](https://www.spokestack.io/create) one. The model download URL can be found in your [account settings](https://www.spokestack.io/account/settings).
+
+
+## Dependencies
+There are also some dependencies that should be installed before using this integration. Audio input is controlled by `pyaudio`, which has [PortAudio](http://www.portaudio.com/) as a dependency. You can install PortAudio from command line:
+
+#### Debian/Ubuntu
+
+```shell
+sudo apt-get install portaudio19-dev
+```
+
+
+#### Alpine
+
+```shell
+apk install portaudio
+```
+
+#### PyAudio
+
+```shell
+pip3 install pyaudio
+```
+
+#### TFLite Runtime
+In addition, you will need a way to run TFLite models. The full [TensorFlow](https://www.tensorflow.org/) package will work, but the recommended approach is the TFlite Runtime. This is hardware specific, so you will want to follow the [installation instructions](https://www.tensorflow.org/lite/guide/python#install_just_the_tensorflow_lite_interpreter).
```

Comment:
TFLite is not available for the larger part of our userbase.

Reply:
Is the full Tensorflow acceptable though? I can rewrite with instructions similar to this [integration](https://www.home-assistant.io/integrations/tensorflow/). Also, would you like me to fully remove the TFLite reference? There are several [wheels](https://github.com/google-coral/pycoral/releases/) available, but I'm not sure if there is enough overlap to still be useful for your users. The TFLite Runtime could provide a lightweight method for those who can install the wheels though.

Reply:
We have the full tensorflow available, that is prebuilt, the wheels available from google on PyPI are useless, as they are not compatible with musl based systems.